src/utils/operation.ts

```diff
@@ -16,7 +16,7 @@ export interface ITransaction {
   opts?: {
     gasLimit?: number
     gasPrice?: number
```

Comment:
should gasPrice b e also a sting ? it might not fit into 53 bit number as well..

Reply:
Does it related to this PR?
Looks like there is no use of `gasPrice` in `joinAndQuit/plugin.ts`

Reply:
Seem not related, but neither is the change on the next line :)

In any case, to be consistent with most of the usage in arc.js, we would use a BN, not.a string (and not a number) here.

Reply:
The change is necessary if I change to "`toString`" in `plugin.ts`.

Reply:
gasPrice cannot be a number as it is not fit into 53 bits .
it might be BN though